contracts/OysterPearl.sol

```diff
@@ -47,7 +47,7 @@ contract OysterPearl {
   function OysterPearl() public payable {
     director = msg.sender;
     name = "Oyster Pearl";
-    symbol = "PRL";
+    symbol = "PRL2";
     decimals = 18;
     saleClosed = true;
```

Comment:
There's no reason for leaving this in if there is no more sale functionality?

Reply:
If we decide to remove this we should also remove line 128 and 13.